onnxruntime/core/providers/cpu/signal/window_functions.cc

```diff
@@ -110,7 +110,7 @@ Status BlackmanWindow::Compute(OpKernelContext* ctx) const {
   return create_cosine_sum_window(ctx, data_type_, a0, a1, a2, is_periodic_);
 }
 
-static inline double hz_to_mel_scale(double hz) { return 2595 * std::log10(1 + hz / 700); }
+static inline double hz_to_mel_scale(double hz1) { return 2595 * std::log10(1 + hz1 / 700); }
 
```

Comment:
Can you explain how `hz` clashes with an AIX variable? Not clear to me how an argument name could do that.

Reply:
Hi @skottmckay 
In AIX /usr/include/sys/m_param.h, has 
#define __hz    HZ      /* Berkeley uses lower case hz      */
#define hz      __hz

This file is included in window_functions.cc through below include path:
/usr/include/sys/m_param.h <- /usr/include/sys/context.h <- /opt/freeware/lib/gcc/powerpc-ibm-aix7.3.0.0/10/include-fixed/sys/signal.h <- /opt/freeware/lib/gcc/powerpc-ibm-aix7.3.0.0/10/include-fixed/sys/wait.h <- /opt/freeware/lib/gcc/powerpc-ibm-aix7.3.0.0/10/include-fixed/stdlib.h <-  /opt/freeware/lib/gcc/powerpc-ibm-aix7.3.0.0/10/include/c++/cstdlib <- /opt/freeware/lib/gcc/powerpc-ibm-aix7.3.0.0/10/include/c++/bits/stl_algo.h <- /opt/freeware/lib/gcc/powerpc-ibm-aix7.3.0.0/10/include/c++/algorithm <- include/onnxruntime/core/common/common.h <- core/providers/cpu/signal/window_functions.h

Reply:
Ah ok. Not the greatest choice of a #defne name.

Would be great to capture this reason for using 'hz1' in a comment so it's not accidentally changed in the future. As we don't have an AIX machine to build on our CIs would not fail if that happened.

Reply:
Yes - Will add that in comment. 
// 'hz' is a #define in AIX, hence using hz1. 
static inline double hz_to_mel_scale(double hz1) { return 2595 * std::log10(1 + hz1 / 700); }

One comment in first usage is sufficient right.

Reply:
I'd do it in both places. I wouldn't assume someone making a change in CreateMelWeightMatrix to have seen the comment in a completely separate function.

Reply:
Ok - Will do at all the places, and reply you.

Reply:
Hi @skottmckay  Pushed the changes with comment where hz1 is used.